sermant-plugins/sermant-flowcontrol/flowcontrol-common/src/main/java/com/huawei/flowcontrol/common/core/match/operator/CompareOperator.java

```diff
@@ -33,7 +33,8 @@
 import java.util.logging.Logger;
 
 /**
- * 比较, 格式如下, 存在有两个符合和一个符号的场景 请求路径包含/b 且    d=10 e != 5 f >= 11 - apiPath: contains: /b headers: d: compare: =10 e:
+ * compare in the following format： There are two matches and one symbol that the scene request path contains/b and
```

Comment:
There are two  scenes： two symbols and one symbol

Reply:
done